### PR TITLE
feat: support Mellow badge

### PR DESCRIPTION
**Pull request by ssbarbee**
What is missing is the correct mapping to BE type. Once @ioanaroceanu2000 has this we can easliy update the FE map

### CODE REVIEW

**Comment by ioanaroceanu2000** [comment]
let's review this after the backend integration, looks like some data is missing

File: src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.mocks.ts [hunk 1]
```diff
@@ -59,6 +59,10 @@ export const season1Badges: CollectionBadge[] = [
   {
     variant: 'dryIce',
   },
+  {
+    variant: 'mellowLpVault',
+    achievedAt: new Date('02/02/2022').valueOf(),
```
Comment (ioanaroceanu2000) [inline src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.mocks.ts:64]: what if the user didn't achieve this? where do we verify this against subgraph data?

Comment (ssbarbee) [reply src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.mocks.ts:64]: this is storybook. Not used anywhere in Application. This is only for the purpose of demo. check the mappers in getSeasonBadges for the actual logic

Comment (ioanaroceanu2000) [reply src/routes/Profile/ProfilePageWalletConnected/ProfilePageWalletConnected.mocks.ts:64]: ok, got it

**Review by ioanaroceanu2000 — APPROVED** [review]

**Pull request merged**